### ISSUE TITLE
List view for Judgement Entities

### ISSUE THREAD

**Issue by oakmac** [threatgrid/ctia-ui#2]
User needs to be able to see a table of all the Judgements in a CTIA instance.

What column headers should we have for Judgements?

**Comment by oakmac** [comment]
Column headers:
- Observable
- Disposition
- Confidence
- Severity

The rest can be in the "expanded view" row.

**Comment by oakmac** [comment]
Attached is the current state of the Judgements table:

![judgements-table](https://cloud.githubusercontent.com/assets/98485/20020200/11d3a060-a29e-11e6-90df-e21665189a79.png)

**Comment by oakmac** [comment]
Disposition column should display the Disposition name, not the number.

**Comment by craigbro** [comment]
We should display the reason.

We also need an expanded view.  That expanded view should show the remaining fields.  It should also show any indicators associated with the Judgement, presenting the title/short_descirption of the indicator, and a link.

**Comment by craigbro** [comment]
The row needs to show the type of the observable as well.

**Comment by craigbro** [comment]
The search bar can be hooked up to the following URL:

```
http://tenzin-beta.amp.cisco.com/ctia/judgement/search?query=foo
```

The contents of the bar are put into the "query" parameter.  Additional `before/after` params can specify date ranges, and `offset/limit` provider controls for pagination.

See response headers for X-Total-Hits other pagination data.

**Comment by oakmac** [comment]
Expanded row screenshot:
![judgements-table-expanded-row](https://cloud.githubusercontent.com/assets/98485/20123478/fd0100ca-a5e4-11e6-9f7f-663ca474826f.png)

**Comment by pxninja** [comment]
IMPORTANT
• Observables
• Disposition
• Reason
• Indicator
• Valid Time
• Severity
• Confidence
• Source

NOT IMPORTANT (meta data)
• Everything else

**Comment by pxninja** [comment]
![expanded-row-02](https://cloud.githubusercontent.com/assets/17751642/20188774/cbb078fa-a73f-11e6-82f8-ce1323366d0b.png)

The top wireframe is a literal translation of my last comment. The bottom wire is a UX translation. Let's hop on a call to discuss if another iteration is needed, or we can move forward with one of the above.

**Comment by craigbro** [comment]
Like the bottom.  Tempted to take source out of the top bar to make more room for indicator title.

**Comment by craigbro** [comment]
Also, Dean suggested we put the soruce of the indicator in their display in the expended view

**Comment by pxninja** [comment]
<img width="1249" alt="screen shot 2016-11-14 at 4 51 03 pm" src="https://cloud.githubusercontent.com/assets/17751642/20286086/b403c006-aa8a-11e6-9eca-df9931f56b12.png">
This is now committed to the UI Sandbox.

**Comment by saintx** [comment]
I drafted up a rough sketch of what I think the top level columns could be in these entity tables, and put it into the wiki:
- https://github.com/threatgrid/wiki/wiki/IROH-Entity-Tables

If we use these columns in the top level entity tables, it would let a user crawl the whole object graph from sighting all the way to COAs without going into the expanded view.

I omitted the Incident entity, because the GUI for Incidents requires special attention, IMO.  It's not just another node in the graph, in my opinion, but really the intersection of all of the other data with respect to the afflicted customer organization.
